packages/editor/src/components/collapsible-block-toolbar/style.scss

```
@@ -64,8 +64,9 @@
 		// Move up a little to prevent the toolbar shift when focus is on the vertical movers.
 		@include break-small() {
 			&:not(.is-horizontal) .block-editor-block-mover__move-button-container {
+				height: 40px;
```

```suggestion
				height: $grid-unit-50;
```

Wouldn't it be better not to have a hardcoded value?